addon/components/cf-field/input/table.js

```diff
@@ -6,6 +6,11 @@ import saveDocumentTableAnswerMutation from "ember-caluma/gql/mutations/save-doc
 import { inject as service } from "@ember/service";
 import { ComponentQueryManager } from "ember-apollo-client";
 
+// TODO check why babel polyfill didn't pick this up
```

Comment:
I would recommend adding a real polyfill.

```js
if (!Array.prototype.flatMap) {
  Array.prototype.flatMap = function() {
    return Array.prototype.map.apply(this, arguments).flat(1);
  };
}
```

[Source](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap#Polyfill)

Reply:
I think the best would be to make sure that babel polyfills this. [This post](https://stackoverflow.com/a/55420761) mentions that @babel/preset-env should in theory be able to do it. We might have to merge #195 first.

Reply:
True. But you will have to update your code as well when you do as the syntax is different. But yeah. As you will have to remove the polyfill you will have to touch the code anyway.

Reply:
If I understand [this issue](https://github.com/babel/babel/issues/9749) correctly we need core-js@3 which only works with babel@7 AFAIK. So we should try to merge #195 before we add some dirty solution which this definitely is..

Reply:
If I understand [this issue](https://github.com/babel/babel/issues/9749) correctly we need core-js@3 which only works with babel@7 AFAIK. So we should try to merge #195 before we add some dirty solution which this definitely is..

Reply:
@fkm I just pushed this PR to illustrate the problem - if we can get the polyfill for this to work in the next couple of days, we can close this PR without ever merging it.